resources/js/features/root/components/navbar/NavigationSection.tsx

```diff
@@ -0,0 +1,25 @@
+import {AccessBarrier, AccessBarrierProps} from "components/utils";
+import {Component, For, Show, splitProps} from "solid-js";
+import {NavigationItem, NavigationItemProps} from "./NavigationItem";
+
+export interface NavigationSectionProps
+  extends Omit<AccessBarrierProps, "children" | "queries" | "Fallback" | "Error" | "Pending"> {
+  title?: string;
+  items: NavigationItemProps[];
+}
+
+const noop = () => null;
+
+export const NavigationSection: Component<NavigationSectionProps> = (props) => {
+  const [localProps, accessBarrierProps] = splitProps(props, ["items", "title"]);
+  return (
+    <AccessBarrier {...accessBarrierProps} Fallback={noop} Error={noop} Pending={noop}>
```

Comment:
Jakiś spinner to by się mam wrażenie przydał w tym menu, tylko ten mały pewnie. Bo się ładuje, doładowują się sekcje, a nie widać nic.

A co do reszty, to nie idzie bez nulla?

Reply:
AccessBarrier ma swoje założenia, które tutaj się nie aplikują. Mikołaj nie chciał żadnych efektów w menu, po prostu miały znikać/zmieniać się. Ale mogę pomyśleć o jakimś efekcie wizualnym na zmianę. A noop to jedyny sposób, z definicji noop nic nie robi i nic nie zwraca (tutaj musi, bo to komponent, normalnie noop to () => void, ale jako komponent średnio pasuje :D

Reply:
Coś powinno być, bo ta sekcja się czasem pojawia z opóźnieniem więc przez chwilę wygląda niepoprawnie. No ale to można potem.